index.html

```diff
@@ -295,358 +321,405 @@ <h3 id="w3c-coordination">W3C Groups</h3>
+          This charter is written in accordance with the <a href="https://www.w3.org/Consortium/Process/#Votes">W3C Process Document (Section 3.4, Votes)</a> and includes no voting procedures beyond what the Process Document requires.
+        </p>
+      </section>
 
-        <dl>
-          <dt>
-            <a href="https://www.iso.org/committee/45316.html">ISO/IEC JTC-1/SC-29 WG 11 Moving Picture Experts Group
-            (MPEG)</a>
-          </dt>
 
-          <dd>This group is developing standards for coded representation of digital audio and video, including
-          MPEG-4.</dd>
 
-          <dt>
-            <a href="https://www.smpte.org/">Society of Motion Picture and Television Engineers (SMPTE)</a>
-          </dt>
+      <section id="patentpolicy">
 
-          <dd>This organization was founded to advance theory and development in the motion imaging field. SMPTE
-          produced extensions to TTML1 that are part of SMPTE-TT.</dd>
+        <h2>
+          Patent Policy
+        </h2>
+        <p>
+          This Working Group operates under the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a> (Version of 15 September 2020). To promote the widest adoption of Web standards, W3C seeks to issue Web specifications that can be implemented, according to this policy, on a Royalty-Free basis.
 
-          <dt>
-            <a href="https://www.ebu.ch/home">European Broadcasting Union (EBU)</a>
-          </dt>
+          For more information about disclosure obligations for this group, please see the <a href="https://www.w3.org/groups/wg/timed-text/ipr">licensing information</a>.
+        </p>
 
-          <dd>The EBU is an association of national broadcasting organizations, facilitating the exchange of
-          audiovisual content. The EBU has developed EBU-TT, an XML-based format for use in subtitling production and
-          exchange, and EBU-TT-D for use in subtitle distribution, both of which are constrained and extended variants
-          of TTML1.</dd>
+      </section>
 
-          <dt>
-            <a href="https://www.dvb.org/groups/TM">DVB Project: Technical Module (DVB-TM)</a>
-          </dt>
 
-          <dd>The DVB Project develops specifications for digital television systems, which are turned into standards
-          by international standards bodies such as ETSI or CENELEC. It provides a conduit to other relevant
-          standardisation activities including MPEG for the purpose of meeting the objectives of the DVB Project.</dd>
 
-          <dt>
-            <a href="https://www.atsc.org/">Advanced Television Systems Committee (ATSC)</a>
-          </dt>
+      <section id="licensing">
+        <h2>Licensing</h2>
+        <p>This Working Group will use the <a href="https://www.w3.org/Consortium/Legal/copyright-documents">W3C Document license</a> or <a href="https://www.w3.org/Consortium/Legal/copyright-software">W3C Software and Document license</a> for all its deliverables.</p>
```

Comment:
This has lost the sense of "WG is able to choose" but gained the sense of "it must be one or the other". Can we keep both senses?

```Suggestion
       <p>This Working Group will use either the <a href="https://www.w3.org/Consortium/Legal/copyright-documents">W3C Document license</a> or the <a href="https://www.w3.org/Consortium/Legal/copyright-software">W3C Software and Document license</a> for each of its deliverables, to be chosen by the Working Group.</p>
```

Reply:
ah, sorry. wrong editing from template. we need to pick one from two licenses by default.
(something interesting, but software and document license does not cover all items in document license...)
https://w3c.github.io/charter-drafts/charter-template.html#licensing

so, would keep the text from 2019 version?

Reply:
I remember arguing about that 2019 text before - it seems to suggest that we will choose only one licence and apply it to all our documents, but that's not what we want. We should be able to choose on a document-by-document basis.

Reply:
I can copy the 2019 text here, but we (or chairs?) are better to raise an issue to the template for initiating discussion also if we copy here.